src/app/components/chat/chat-detail/chat-detail.component.ts

```diff
@@ -111,7 +92,19 @@ export class ChatDetailComponent implements OnInit {
     });
   };
 
-  sendMessage = () => {
-    console.log(this.msgInput);
+  onSendMessage = () => {
+    const { uid } = this.authSvc.authInfo$.value,
+      { chatGroupId, msgInput, fbSvc, afs } = this;
+
+    const messageData: MessageI = {
+      id: afs.createId(),
```

Comment:
We can leave this out since id is optional, and instead create it in the ChatService
```suggestion
```